sonar-project.properties

```diff
@@ -1,17 +1,20 @@
-sonar.projectKey=sap_cloud-commerce-spartacus-storefront
-sonar.projectName=Cloud commerce spartacus storefront
+sonar.host.url=https://sonar.tools.sap
 
+sonar.projectKey=pwa-storefront-toolset
+sonar.projectName=pwa-storefront-toolset
+sonar.branch.name=release/5.0-next.2
```

Comment:
2 questions:

1 - related to where I'm pointing here. I don't think we should specify a specific branch name here as we would need to change this many times. Should we just run sonar on the 'default' branch, therefore not specifying a branch

Reply:
What do you think?

Reply:
Thing is it will override the develop sonar scan no? Like we will run the sonar scan nightly for develop but if we then run this 5.0 branch for release, then it will show the 5.0 branch issues on the develop branch (default) which can be confusing?

Reply:
I think each release branch should have its own sonar scan branch too but maybe im thinking of it wrong.
When this PR gets merged into develop, I can set develop there (or leave it empty cause it'll default to that)